WireAPI/Sources/WireAPI/APIs/ConversationsAPI/APIModel/QualifiedConversationMember.swift

```diff
@@ -0,0 +1,81 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+struct QualifiedConversationMember: Decodable, ToAPIModelConvertible {
+
+    struct Service: Decodable, ToAPIModelConvertible {
+        let id: UUID
+        let provider: UUID
+
+        func toAPIModel() -> WireAPI.Service {
+            // TODO: in WireRequestStrategy the object had also UUIDs
+            // but we want to unify the model classes. So does it work with Strings like that?
```

Comment:
This is an open question. @typfel I think you added the `WireAPI.Service` with string, do you have some opinion or knowledge here?

Reply:
So the ID of an service is a UUID according to Swagger so we can definitely change it to that. I guess I was porting the service definition from android where they don't decode the id into a UUID but refer to it as an opaque string.

I think the reason why iOS client historically has been decoding UUIDs is because they can be persisted more efficiently.

Reply:
Thanks @typfel, updated with b0e55e5c26bce46a9614970acce6c6c1ff1b714a